app/views/users/show.html.erb

```diff
@@ -1,3 +1,47 @@
-<% 3.times do %>
-  <h1>Users show view</h1>
-<% end %>
\ No newline at end of file
+<main>
+  <section class="users-section">
+    <h2>User information</h2>
+    <article class="users-info">
+      <div class="user-card">
+        <div class="image-container">
+          <img src="<%=@user.photo || image_url('user_icon.png') %>" alt="Image of user <%=@user.name %>" title="<%=@user.name %>" class="user-image">
+        </div>
+        <div class="name-counter"
+          <h4><%= @user.name %></h4>
+          <p>Number of posts: <%= @user.post_counter %></p>
+        </div>
+      </div>
+    </article>
+
+    <article class="users-bio">
+      <p>User bio:  <%= @user.bio %></p>
+    </article>
+
+    <article class="users-posts">
+      <% if @user.last_three_post.empty? %>
+        <h3>Currently there are not posts in the database for this user</h3>
+      <% else %>
+        <% @user.last_three_post.each do |post| %>
+          <div class="posts-section">
+            <%= link_to user_post_path(@user.id, post.id) do %>
+              <div class="posts-card"> 
+                <h4>Post #: <%= post.id %> - <%= post.title %></h4>
+                <% end %>
+                <p class="post-text"><%= post.text %></p>
+                <p class="post-comments-likes">
+                  Comments: <%= post.comments_counter %> - Likes: <%= post.likes_counter %>
+                </p>
+              </div>
+            
+            
+          </div>
+       <% end %>
+     <% end %>
+    </article>
+
+    <%= link_to user_posts_path(@user) do %>
+      <button class="see-all-button">See all posts</button>
+    <% end %>
+    
+  </section>
```

Comment:
- [ ] Kindly make sure the app/views/users/show.html.erb view page match the given [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_user_page.png) design as required for the project. Right now, no styling has been added.

Reply:
Styling is nice to have but focus on making sure everything functions first. (Note: Ignore login logic, you would be using Devise in coming milestone)